.pre-commit-config.yaml

```diff
@@ -17,7 +17,7 @@ repos:
       - id: check-yaml
       - id: check-toml
       - id: check-ast
-      - id: check-added-large-files
+      # - id: check-added-large-files
```

Comment:
```suggestion
      - id: check-added-large-files
        exclude: |
          (?x)(
            ^images/
          )
```

Reply:
You can also upload the large image via the GitHub issue page and insert the image URL rather than put the image file in the repo.

Reply:
I upload the gif via the GitHub issue page and update the image URL in README, but also I will update the `check-added-large-files` with exclude.